### PR TITLE
Re-add yang_json2xml lookup plugin

### PR DESCRIPTION
**Pull request by ganeshrn**
Add yang_json2xml lookup plugin which validated input json configuration instance and convert it to xml

### CODE REVIEW

File: tests/test.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+#- import_playbook: text_parser/test.yml
+#- import_playbook: textfsm/test.yml
+#- import_playbook: command_parser/test.yml
+#- import_playbook: textfsm_parser/test.yml
+#- import_playbook: json_template/test.yml
+#- import_playbook: vlan_compress/test.yml
+#- import_playbook: vlan_expand/test.yml
+#- import_playbook: netcfg_diff/test.yml
+#- import_playbook: interface_range/test.yml
+#- import_playbook: interface_split/test.yml
```
Comment (trishnaguha) [inline tests/test.yml]: Please uncomment the other tests.

File: tests/yang_json2xml/yang_json2xml/meta/main.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - ../../../network-engine
```
Comment (trishnaguha) [inline tests/yang_json2xml/yang_json2xml/meta/main.yaml]: This is not needed anymore as the test role is imported in tasks/main.yaml

**Review by trishnaguha — CHANGES_REQUESTED** [review]

File: tests/yang_json2xml/yang_json2xml/meta/main.yaml (hunk 2 above)
Comment (ganeshrn) [reply tests/yang_json2xml/yang_json2xml/meta/main.yaml]: fixed

File: tests/test.yml (hunk 1 above)
Comment (ganeshrn) [reply tests/test.yml]: Fixed

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**